src/components/pages/Home/RenderHomePage.js

```diff
@@ -95,19 +96,19 @@ function RenderHomePage(props) {
       )
     )
       .then(res => {
-        window.localStorage.setItem('Admin', res.data.is_admin);
+        window.localStorage.setItem('role', res.data.role);
         setHrfUserInfo(res.data);
-        setSavedCases(res.data.case_bookmarks);
-        setSavedJudges(res.data.judge_bookmarks);
+        // setSavedCases(res.data.case_bookmarks);
```

Comment:
Do these still need to be here and commented out?

Reply:
Ah yes, I left there because the saved cases/judges are just being refactored so it currently breaks the app, yet I didn't want to remove it so we know where it was being handled in case its just a naming change or something similar